app/js/lib/foxbox.js

```diff
@@ -10,6 +10,11 @@ import FoxboxDb from './foxbox-db';
 const settings = new FoxboxSettings();
 const db = new FoxboxDb();
 
+const priv = {
```

Comment:
Ha, I've remembered why we called it ```priv``` - ```private``` is reseved keyword in strict mode [1] :)

Do you want me to rename it to something like ```privateMembers``` then or just leave ```priv```?

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Paving_the_way_for_future_ECMAScript_versions.